common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/OfflineCustomerInfoCalculator.kt

```diff
@@ -78,6 +78,7 @@ class OfflineCustomerInfoCalculator(
         return minPurchaseDate?.let { ISO8601Utils.format(Date(minPurchaseDate)) }
     }
 
+    @SuppressWarnings("ForbiddenComment")
```

Comment:
What I've done sometimes is change the `TODO:` with just `TODO` or `WIP`... NABD though